### PR TITLE
renamed property to resolve name conflict

### PR DESCRIPTION
**Pull request by SimonSchick**

### CODE REVIEW

**Comment by connor4312** [comment]
We could follow the naming convention that all private/protected members be prefixed with `_`

**Comment by SimonSchick** [comment]
If we actually enforce this policy **everywhere** and not just on individual repos, I'm fine with it.

If you can find a rule that can **strictly** enforce this that would be fantastic, like https://palantir.github.io/tslint/rules/variable-name/ but force it for private members.

File: src/ModelStore.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { gc } from './util/GarbageCollector';
 import { ModelState } from './util/State';
 
 export interface IModelRecord {
-    state: ModelState;
+    state: ModelState; // tslint:disable-line
```
Comment (SimonSchick) [inline src/ModelStore.ts]: This works, the tslint comment is redundant tho.

**Comment by connor4312** [comment]
Microsoft added a rule for function names

**Comment by SimonSchick** [comment]
Unfortunately that's not really going to help us in this case.

**Comment by connor4312** [comment]
We can rename protected and private methods. It would also be simple, probably, to add a `property-name` rule based on that.

**Pull request merged**